Name: Hide Backup
Version: 2.2.4-1
Author: webOS Archive
Description: Hide Backup icon from Launcher

--- .orig/usr/palm/applications/com.palm.app.backup/appinfo.json
+++ /usr/palm/applications/com.palm.app.backup/appinfo.json
@@ -4,6 +4,8 @@
 	"main": "index.html",
 	"id": "com.palm.app.backup",
 	"icon": "icon.png",
+	"removable": true,
+	"visible": false,
 	"splashicon": "icon-256x256.png",
 	"keywords": ["Settings", "Preferences"]
 }
